examples/data-objects/prosemirror/src/fluidCollabManager.ts

```diff
@@ -181,7 +184,7 @@ export class FluidCollabManager extends EventEmitter implements IRichTextEditor
             run(state, dispatch) {
                 const { empty, $from, $to } = state.selection;
                 let content = Fragment.empty;
-                if (!empty && $from.sameParent($to) && $from.parent.inlineContent) {
+                if (Boolean(empty) && Boolean($from.sameParent($to)) && Boolean($from.parent.inlineContent)) {
```

Comment:
Are there prosemirror types we can pull in?  Looks like there might at least be prosemirror-menu types.

Looks like you unintentionally flipped the empty check here too

Reply:
Yeah, I got sloppy. Thanks for catching this. I hadn't considered types -- I'll look into that.

Reply:
I updated the types packages, but I've not figured out how to use them properly.